queries/nginx/highlights.scm

```diff
@@ -0,0 +1,61 @@
+(comment) @comment @spell
+
+(value) @variable
```

Comment:
I am wondering if this would make more sense as `@string`, if it would be more similar to like an `ini` type file?

Reply:
If someone wants my comment to re-review PR - idk 

Please share your wisdom what is better - `@variable` or `@string`? 🙇